CHANGES.txt

```diff
@@ -1,6 +1,37 @@
 Codetools Changelog
 ===================
 
+Release 4.3.1
+-------------
+
+Released : 23 March 2021
+
+Fixes
+
+* Add manifest file (#36)
+
+* Exclude unsupported submodules on Py3 (#39)
+
+Build
+
+* Use etstool module in CI setup (#41)
+
+* Update CI (#44)
+
+* Disable macOS builds on Travis CI (#46)
+
+Maintenance
+
+* Replace deprecated rich_compare kwarg (#43)
+
+* Replace use of apptools sweetpickle
```

Comment:
Sorry for not being clear about the 4.3.1 release. I was hoping on only including this fix/PR in the 4.3.1 release - and not all of the changes that have been made on the main branch since 4.3.0.

That might be slightly complicated because I don't see a maintenance branch for the 4.3.x releases. Let's discuss this today at the standup and we can see what the best way ahead is.